### PR TITLE
Adding a former companies directory

### PR DESCRIPTION
**Pull request by clone1018**
It also removes present-tense references to Gratipay.

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -1,80 +1,84 @@
 <!DOCTYPE html>
```
Comment (clone1018) [inline _layouts/default.html]: Somehow this file got autoformatted. Let me know if this is a problem!

File: img/logos/salalem.svg [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" standalone="no"?>
```
Comment (clone1018) [inline img/logos/salalem.svg]: Converted their logo into a SVG since it was missing.

File: _layouts/default.html (hunk 1 above)
Comment (galuszkak) [reply _layouts/default.html]: I don't see this as an issue.

File: img/logos/salalem.svg (hunk 2 above)
Comment (galuszkak) [reply img/logos/salalem.svg]: Thanks!

**Pull request merged**